src/pybind/mgr/rook/generate_rook_ceph_client.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+python3 -m venv .venv
+
+trap "rv=\$?; rm -rf .venv; exit \$rv" EXIT
+
+. .venv/bin/activate
+pip install -e git+https://github.com/sebastian-philipp/rook-ceph-client-python.git#egg=rook-ceph-client
+
+curl https://raw.githubusercontent.com/rook/rook/master/cluster/examples/kubernetes/ceph/common.yaml | generate-model-classes --create-package  - rook_ceph_client
```

Comment:
In my experience, GitHub is not 100% reliable, so I assume this line will fail from time to time.